scripts/all-tests.sh

```diff
@@ -13,7 +13,7 @@ set -e
 # Violations thresholds for failing the build
 export LOWER_PYLINT_THRESHOLD=1000
 export UPPER_PYLINT_THRESHOLD=5900
-export ESLINT_THRESHOLD=5700
+export ESLINT_THRESHOLD=5561
```

Comment:
nice

Reply:
I can't claim all those fixes. I think I reduced the number by 1. :) But good to keep cranking this threshold down.